### ISSUE TITLE
Support for tables without outer pipes

### ISSUE THREAD

**Issue by arteme** [anton-dev-ua/PipeTableFormatter#2]
Pipe table formatter plugin is very neat. It would be very nice if it were able to support tables without outer pipe characters. This is specifically useful when writing parametrized Spock test cases like:
```groovy
class Math extends Specification {
    def "maximum of two numbers"(int a, int b, int c) {
        expect:
        Math.max(a, b) == c

        where:
        a | b | c
        1 | 3 | 3
        7 | 4 | 4
        0 | 0 | 0
    }
}
```

If I were to reformat the where-clause table with Pipe Table Formatter, it would wrap the table in pipes, which is not a valid parameter table anymore.

**Comment by anton-dev-ua** [comment]
Thanks for your comment!  I think this will be a useful feature. Will take a look.

**Comment by anton-dev-ua** [comment]
implemented in 0.3.1
